set_env_var.sh

```diff
@@ -11,3 +11,5 @@ export AIRFLOW_WEBSERVER_PID=$PROJECT_HOME/airflow-webserver.pid
 export SCRAPER_FILE_PREFIX=blog_articles
 export WEBAPP_DIR=$PROJECT_HOME/webapp
 export WEBAPP_PLOT_DIR=$WEBAPP_DIR/templates/static/plotly
+export PYTHONPATH=$PROJECT_HOME:$PYTHONPATH
+export AIRFLOW_START_DATE=`date +%Y-%m-%d`
```

Comment:
setting up the start date as today's date may have some unwanted effect: namely if you schedule tasks that take a long time the dag may never actually run. It's not encouraged practice in Airflow. Consider using a fixed start date or adding a buffer (like 1/2 days before today's date)

Reply:
1. Not sure what you meant by it will never run. Why would the dag never run if a process takes a long time?
2. It is true that the start date should not be dynamic in the `dag` file, however, the `set_env_var.sh` file is merely a configuration which can be changed. Generally, you will have multiple configuration file for testing, staging and deployment. This avoids having to change the dag file every single time we deploy, which I have done multiple time.

Reply:
Just a matter of [code quality](https://airflow.readthedocs.io/en/latest/faq.html?highlight=start_date), besides having a variable parameter impacts reproducibility.